src/lib/response-triggers/help.ts

```diff
@@ -236,7 +250,47 @@ export default class Help extends Trigger {
 	createRoleConfigHelpMessage(message: MessageObject) : string {
 		if (this.config.checkRole(message.personId, Role.Admin) || this.config.checkRole(message.personId, Role.Config)) {
 			return `### How to configure role config values 🗞️!
-1. To get/edit/refresh/delete user roles, type \`@${constants.botName} roleconfig ${Object.values(ConfigAction).join('|')} {@User} {role}\`
+1. To get/edit/refresh/delete user roles, type \`@${constants.botName} ${Command.RoleConfig} ${Object.values(ConfigAction).join('|')} {@User} {role}\`
+1. I will respond in the room you messaged me in.`;
+		} else {
+			return this.createDefaultHelpMessage();
+		}
+	}
+
+	createLocationConfigHelpMessage(message: MessageObject) : string {
+		if (this.config.checkRole(message.personId, Role.Admin) || this.config.checkRole(message.personId, Role.Config)) {
+			return `### How to configure location config values!
+1. To get/edit/delete locations, type \`@${constants.botName} ${Command.LocationConfig} ${Object.values(LocationAction).join('|')} {location}\`
+1. I will respond in the room you messaged me in.`;
+		} else {
+			return `### How to get location values!
+1. To get a list of locations, type \`@${constants.botName} ${Command.LocationConfig} ${LocationAction.Get}\`
+    * To configure locations, please ask an admin.
+1. I will respond in the room you messaged me in.`
+		}
+	}
+
+	createUserLocationHelpMessage(message: MessageObject) : string {
+		if (this.config.checkRole(message.personId, Role.Admin) || this.config.checkRole(message.personId, Role.UserLocation)) {
+			return `### How to configure user location values!
+1. To get user locations for yourself or others, type \`@${constants.botName} ${Command.UserLocation} ${LocationAction.Get} me|all|unset|@User\`
+1. To set user locations, type \`@${constants.botName} ${Command.UserLocation} ${LocationAction.Set} {location} me|@User1 @User2\`
+1. To delete user locations, type \`@${constants.botName} ${Command.UserLocation} ${LocationAction.Delete} me|@User1 @User2\`
+1. I will respond in the room you messaged me in.`
+		} else {
+			return `### How to config your user location value!
+1. To get user locations for yourself or others, type \`@${constants.botName} ${Command.UserLocation} ${LocationAction.Get} me|all|unset|@User\`
+1. To set your user location, type \`@${constants.botName} ${Command.UserLocation} ${LocationAction.Set} {location} me\`
+	* To bulk configure user locations, please ask an admin.
```

Comment:
Bulk configure feels like weird phrasing here, considering non-admins can't configure anyone but themselves - they don't need to bulk configure their own user location.

Reply:
I'm not sure what you're suggesting here, the fact that users can't bulk configure locations for others is exactly why this is here.